RoutesApp_ios/modules/routelist/views/controllers/RouteListFilterViewController.swift

```diff
@@ -0,0 +1,74 @@
+import UIKit
+
+class RouteListFilterViewController: UIViewController {
+
+    @IBOutlet var resetButton: UIButton!
+    @IBOutlet var tableView: UITableView!
+    @IBOutlet var doneButton: UIButton!
+    @IBOutlet var filterTitleLabel: UILabel!
+
+    var viewModel: RouteListViewModel
+
+    init(viewModel: RouteListViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
is this error necessary?

Reply:
I just removed the unnecessary fatal error.